ema_workbench/em_framework/futures_multiprocessing.py

```diff
@@ -104,7 +104,7 @@ def worker(experiment):
     experiment : dict
 
     """
-    global experiment_runner
+    # global experiment_runner
```

Comment:
Why is it not needed anymore? Can it be fully removed?

Reply:
it's the old code and can be removed.

Reply:
```suggestion
```